lambda_function/ci.tf

```diff
@@ -49,6 +49,7 @@ resource "aws_iam_role_policy" "codebuild" {
         "lambda:UpdateFunctionCode",
         "lambda:ListVersionsByFunction",
         "lambda:UpdateAlias"
+        "${var.codebuild_can_invoke_lambda ? ",lambda:InvokeFunction" : ""}"
```

Comment:
not sure if this is possible or I might have to break this out into separate statements and use the dynamic functionality.

Reply:
Testing in sandbox currently